### PR TITLE
POC-06

### PR DESCRIPTION
**Pull request by ben-hwang**

### CODE REVIEW

**Comment by raymondsitu** [comment]
there are a few misc files included in the commit
ds store
gitignore
class path
etc

File: .classpath [hunk 1]
```diff
@@ -13,13 +13,14 @@
 	</classpathentry>
```
Comment (jyoo980) [inline .classpath:13]: Should not commit this file

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/target/
```
Comment (jyoo980) [inline .gitignore:1]: No need to commit this file

**Comment by jyoo980** [comment]
@ben-hwang 

Can you unstage every single file except for the `.aj` file? Also, please comment out any calls to
`updateXSpeed()` and `updateYSpeed()` which are now captured in the aspect.

**Pull request closed without merging**